### PR TITLE
Gui-Final

### PR DESCRIPTION
**Pull request by lashlockbhs**
Gui that works for all our purposes, perfect shapes still needs to be added. This will count as my project for grading, aswell as my other contributions to physics.

### CODE REVIEW

**Review by szetingchen — APPROVED** [review]
simply the worst gui I have ever seen
most inefficient code ever

but it works so lgtm

File: projects/physics/public/script.js [hunk 1]
```diff
@@ -227,52 +227,30 @@ registerOnKeyDown((k) => {
...
-  }
-};
 
 let next = 0;
 let countFrame = 0;
 const drawFrame = (time) => {
   if (time > next && !paused) {
     clear(ctx, width, height);
     for (const shape of objArray) {
+      const density = parseInt(document.getElementById(shape.menu.name + " density").value)
```
Comment (ionisea) [inline projects/physics/public/script.js:251]: This is a weird way to do density/mass, as you demonstrated on friday. While it clearly works, I don't think a dynamic density makes much sense for this specific circumstance. I recommend you open up a window once, when the shape is first drawn, just to change the density, then close it when the the field is deselected.

File: projects/physics/public/style.css [hunk 2]
```diff
@@ -4,39 +4,51 @@ canvas {
 }
 
 #menuHolder {
+  font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
Comment (ionisea) [inline projects/physics/public/style.css:7]: Are we going to use @szetingchen's lexend font here?

**Review by ionisea — CHANGES_REQUESTED** [review]
change or reply to comments with justification and re-request my review

File: projects/physics/public/script.js (hunk 1 above)
Comment (lashlockbhs) [reply projects/physics/public/script.js:251]: I agree that it doesn't make sence to be able to modify density durning the sim, one could argue that there should be no modifications to shape properties by the user during the sim, but its kinda funny to expirement with these type of modifiable properties.

File: projects/physics/public/style.css (hunk 2 above)
Comment (lashlockbhs) [reply projects/physics/public/style.css:7]: we can change font in later lets just get this shit merged.

**Review by ionisea — APPROVED** [review]
In that case lgtm

**Review by gigamonkey — APPROVED** [review]
I had a bit of trouble using the GUI but obviously you've done a ton of interesting work here. Your project grade will be posted in IC soon.

**Comment by lashlockbhs** [comment]
If you have problems with it your prob using it incorrectly sorry.

**Pull request merged**